### ISSUE TITLE
sequence viewer should eliminate newlines when copy-pasting

### ISSUE THREAD

**Issue by yannickwurm** [wurmlab/sequenceserver#174]
all is in the title

(when I paste sequence I copied from there it contains newlines. It shouldn't

**Comment by yeban** [comment]
How will that be helpful?

**Comment by vivekiitkgp** [comment]
@yeban On another note, do you think we should also re-align the copied sequence?

For example,

sequence-viewer
````
...                         STYS YLAT LILV FLIT DFVR
STYS YLAT LILV FLIT DFVR YKPI IVLC GVSI
GSVT FLM MI
````

Copying this will result in
````
STYSYLATLILVFLITDFVR<space here>
STYSYLATLILVFLITDFVRYKPIIVLCGVSI
GSVTFLMMI
````

**Comment by yeban** [comment]
@vivekiitkgp How do you propose to do that? As a matter of fact, how to process any copied text across all browsers regardless of how the text was copied (Ctrl/Cmd-C, Right click -> Copy, Toolbar -> Edit -> Copy). And if newlines can be removed, the issue you raise won't be manifested either.

**Comment by vivekiitkgp** [comment]
If I remove newlines, the pasted sequence will be on single line (may be wrong). What I meant was similar to `gq` in Vim. I like the latter.

**Comment by yannickwurm** [comment]
yes it should be on a single line.

**Comment by yeban** [comment]
@yannickwurm Sequence viewer's copy-paste behaviour is browser default. Thus what the issue suggests will actually be contrary to user's expectations and not the other way round.

If there's a real reason to do this, we could use [this](http://stackoverflow.com/a/17528590) trick and copy without newlines on `Ctrl/Cmd-C` but keep the default behaviour for other ways to copy text (Right click -> Copy, Toolbar -> Edit -> Copy). If not, I suggest this issue be closed.

**Comment by yeban** [comment]
Fixed in f34cda36d905ad63146bf1ed683129081e3af0bb by disabling grouping of residues.
